playbooks/roles/ocim/meta/main.yml

```diff
@@ -10,6 +10,8 @@ dependencies:
     NGINX_PROXY_SERVER_NAME: "{{ OPENCRAFT_DOMAIN_NAME }}"
     NGINX_PROXY_SEND_TIMEOUT: 600s
     NGINX_PROXY_READ_TIMEOUT: 600s
+    NGINX_PROXY_ORIGIN_REGEX: '^(http|https)://.*\.opencraft\.com'
+    NGINX_PROXY_CORS_HEADERS_FOR_RESPONSE_CODES: [413, 500]
```

Comment:
Let's remove `500` from this list for now since I think having 500 responses with no CORS headers is preferable to 500 responses with empty request bodies.